_includes/embed/audio.html

```diff
@@ -0,0 +1,43 @@
+{% assign src = include.src | strip %}
+{% assign title = include.title | strip %}
+{% assign types = include.types | default: '' | strip | split: '|' %}
+
+{% unless src contains '://' %}
+  {%- capture src -%}
+    {% include img-url.html src=src %}
```

Comment:
I noticed that you use `img-url.html` in the embed audio/video, which may be confusing for future maintenance.

How about changing `img-url.html` to `media-url.html` ?
In that case we could also change `page.img_url` to `page.media_url`, `site.img_cdn` to `site.cdn`.

Reply:
It's fine to rename `img-url.html`, but the problem with `page.img_url` is more complicated and should be handled separately, I think.

Currently, `page.img_url` is not being passed to `img-url.html`, so users may specify a path to video/audio without the need to use full address starting from `https`.

I started using the video embed solution and found out that I cannot use `page.img_url` for video files. @MrMurdock11 mentioned this problem in the documentation here: PWA fails to cache chunked data. In general it might be a bad idea to cache all video/audio files. So I was thinking more in the direction of introducing separate variables for audio/video files. In my blog instance, I had to create custom embed code to achieve it, so I can store video files in a separate folder while not using CDN.

Reply:
So do you think `site.img_cdn` can be used by video/audio?

Reply:
`site.img_cdn` allows us to point to a different base URL like `https://my-cdn.com` instead of the blog URL, say `https://my-blog.com`.  So from an embed perspective we are still able to specify a different folder e.g. `{% include embed/video.html src='/video/myvideo.mp4' %}` which will resolve to `https://my-cdn.com/video/myvideo.mp4`.

The problem occurs when we add `page.img_url` to this chain. For example, `https://my-cdn.com/images/video/myvideo.mp4`. Using `page.img_url` forces us to put video/audio files in the same directory as images.

I suppose this could still be excluded from the PWA cache if we host video files under `/images/video`, but it makes it impossible to archive if we don't use a subdirectory.

So to make it easier, it might make sense to introduce separate variables.

Another option might be to allow PWA to cache video/audio files by fixing the problem in the caching logic. I'm not sure at this point what the best practice is for PWA applications in such cases.

Reply:
I get your point. So for now, we can keep `page.img_url` and only change the following filenames and variables:

- Change `_includes/img-url.html` to `_includes/media-url.html`
- Change `site.img_cdn` to `site.cdn`

Reply:
I don't think these changes need to be implemented in this pull request, because these changes will cause fixes in a lot of files that have nothing to do with this feature. I suggest we close this pull request, and then I will create a new one with changes to `img_cdn` and `img-url`.

Reply:
As you wish @MrMurdock11 :)